packages/Checkboxes/src/index.tsx

```diff
@@ -2,41 +2,44 @@ import React, { Fragment, FunctionComponent, useEffect, useState } from "react";
 import uuidv1 from "uuid/v1";
 interface ICheckboxesProps {
   options?: any[] | object;
-  withEffect?: boolean;
-  bool?: boolean;
+  returnBoolean?: boolean;
   onChange: (...args: any) => void;
 }
 const Checkboxes: FunctionComponent<ICheckboxesProps> = ({
-  bool,
+  returnBoolean,
   onChange,
   options,
-  withEffect,
   ...attrs
 }): JSX.Element => {
-  const [data, setData]: any = useState(
-    Array.isArray(options) ? options : [options]
-  );
-  useEffect(() => {
-    if (withEffect) {
-      setData(options);
-    }
-  }, [options, withEffect]);
+  const formatedOptions = Array.isArray(options) ? options : [options];
+
+  const [data, setData]: any = useState(formatedOptions);
```

Comment:
Should be typed with the same type above

Reply:
is typed, in the checkboxes pr